### PR TITLE
Adds Bonsai sprite to the laurelin bonsai

### PR DESCRIPTION
**Pull request by Twomoon-Github**
## About The Pull Request

Adds bonsai sprites for the Laurelin Bonsai.
They will appear out of one of the 4 presets each round (green, pink, yellow, blue)
remove the pixel shift of the laurelin bonsai on the map, as its not working with the new sprites.
bonsai_showcase

## Why It's Good For The Game

the laurelin bonsai had no sprites, it was a placeholder until one get made, well, now we no longer need a placeholder, and we have 4 variants for it.

## Testing

tested, its 37 pixel height for 32 wide, so its not perfect but it doesnt change anything to the game, and it fit withing the 32x by 52y handyman told me.

## Changelog
:cl:
imageadd: added bonsai.dmi
code: changed bonsai.dm/cev_eris.dmm
/:cl:

### CODE REVIEW

File: code/modules/reagents/bonsai.dm [hunk 1]
```diff
@@ -9,8 +9,9 @@
 	name = "Laurelin bonsai"
 	desc = "A small tree, gifted to the club by a previous patron. It subsists off of numerous alcohols, and produces fruits and vegetables in return."
 
-	icon = 'icons/obj/plants.dmi'
-	icon_state = "plant-21" //Placeholder until we can get a proper sprite for them.
+	icon = 'icons/obj/bonsai.dmi'
+	icon_state = "bonsai_1"
+	layer = ABOVE_MOB_LAYER
```
Comment (Firefox13) [inline code/modules/reagents/bonsai.dm:14]: So this means someone can hide under the sprite. **No** please.

File: code/modules/reagents/bonsai.dm [hunk 2]
```diff
@@ -54,4 +55,9 @@
 					"wheat",
 					"potato",
 					"rice")]
-				S.harvest(get_turf(src),0,0,1)
\ No newline at end of file
+				S.harvest(get_turf(src),0,0,1)
+
+/obj/item/reagent_containers/bonsai/New()
```
Comment (Firefox13) [inline code/modules/reagents/bonsai.dm:60]: 25th line, you have New() used twice (this breaks a lot of things). You need to remove this New() and put icon_state into already existing one.

**Review by Firefox13 — COMMENTED** [review]
Please don't half-ass the sprite size, make it proper 32x32. Everything is 32x32 for a good reason.

File: code/modules/reagents/bonsai.dm (hunk 1 above)
Comment (Twomoon-Github) [reply code/modules/reagents/bonsai.dm:14]: it looks dumb when above it

File: code/modules/reagents/bonsai.dm (hunk 2 above)
Comment (Twomoon-Github) [reply code/modules/reagents/bonsai.dm:60]: i dont understand what you mean, this breaks what?

Comment (Twomoon-Github) [reply code/modules/reagents/bonsai.dm:60]: did what you told me in discord, should be ok i hope

File: code/modules/reagents/bonsai.dm (hunk 1 above)
Comment (Twomoon-Github) [reply code/modules/reagents/bonsai.dm]: changed, after checking, being above the bonsai layer is ok

**Comment by Twomoon-Github** [comment]
> Please don't half-ass the sprite size, make it proper 32x32. Everything is 32x32 for a good reason.

fixed the sprite size, now its 32x32 without ruining the sprite. thanks for the help btw.

**Pull request closed without merging**